chapters/synchronous.tex

```diff
@@ -342,17 +342,18 @@ \section{Clock Constructors}\label{clock-constructors}
 \end{lstlisting}\end{synopsis}
 \begin{semantics}
 \firstuse{Rational interval clock}.
-The first input argument, $\mathit{intervalCounter}$, is a clocked component expression (\cref{def:component-expression}) or a parameter expression of type \lstinline!Integer! with \lstinline!min = 0!.
+The first input argument, $\mathit{intervalCounter}$, is a clocked component expression (\cref{def:component-expression}), or a parameter expression of type \lstinline!Integer! with \lstinline!min = 0!.
```

Comment:
I feel like the comma is not necessary here. There are only two items  in the list. Usually, it is used for in lists of three or more items.

Reply:
Ok, removed.